gnome-extra/gnome-software/gnome-software-43.0.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson virtualx xdg
+
+DESCRIPTION="Gnome install & update software"
+HOMEPAGE="https://wiki.gnome.org/Apps/Software https://gitlab.gnome.org/GNOME/gnome-software"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="flatpak +firmware gnome gtk-doc sysprof udev"
+RESTRICT="test" # See TODO below
```

Comment:
Once #27366 is merged, we can enable testing here.

Reply:
Ohh, nice. Thanks.

Reply:
Let's try enabling tests.

Reply:
I see one failing test:

````
8/9 gnome-software:plugins+fwupd / gs-self-test-fwupd       FAIL             3.58s   killed by signal 6 SIGABRT
````

The failure is:

````
Bail out! GsPluginFwupd:ERROR:../gnome-software-43.0/plugins/fwupd/gs-self-test.c:61:gs_plugins_fwupd_func: assertion failed (gs_app_get_update_details_markup (app) == "Some of the platform secrets may be invalidated when " "updating this firmware. Please ensure you have the " "volume recovery key before continuing.\n\nLatest " "firmware release."): ("Latest firmware release." == "Some of the platform secrets may be invalidated when updating this firmware. Please ensure you have the volume recovery key before continuing.\n\nLatest firmware release.")
stderr:
**
GsPluginFwupd:ERROR:../gnome-software-43.0/plugins/fwupd/gs-self-test.c:61:gs_plugins_fwupd_func: assertion failed (gs_app_get_update_details_markup (app) == "Some of the platform secrets may be invalidated when " "updating this firmware. Please ensure you have the " "volume recovery key before continuing.\n\nLatest " "firmware release."): ("Latest firmware release." == "Some of the platform secrets may be invalidated when updating this firmware. Please ensure you have the volume recovery key before continuing.\n\nLatest firmware release.")
````

Do you get the same failure?